### PR TITLE
(Transfer)Wise QuoteV2 Refactor

### PR DESCRIPTION
**Pull request by kewitz**
Resolves https://github.com/opencollective/opencollective/issues/4008
Resolves https://github.com/opencollective/opencollective/issues/3965

### CODE REVIEW

**Comment by patcon** [comment]
Thanks so much @kewitz and OC team 🙌 

Is this now immediately on prod? (If so, I will happily drop my mentioning it as caveat during onboarding of collectives :)

cc @benhylau

File: server/lib/transferwise.ts [hunk 1]
```diff
@@ -95,10 +100,11 @@ export const requestDataAndThrowParsedError = (
   },
   defaultErrorMessage?: string,
 ): Promise<any> => {
-  debug(`calling ${url}`);
+  debug(`calling ${url}: ${JSON.stringify({ data, params: options.params }, null, 2)}`);
   const pRequest = data ? fn(url, data, options) : fn(url, options);
   return pRequest
     .then(getData)
+    .then(tap(data => debug(JSON.stringify(data, null, 2))))
```
Comment (Betree) [inline server/lib/transferwise.ts:107]: That looks good!

**Review by Betree — COMMENTED** [review]
Good improvement!

@patcon I think this has not been deployed in production yet

**Comment by kewitz** [comment]
@patcon this is going to be deployed soon but we're currently having trouble with a non-related dependency build process.

**Pull request merged**